pkg/karmadactl/top/top.go

```diff
@@ -56,3 +65,54 @@ func SupportedMetricsAPIVersionAvailable(discoveredAPIGroups *metav1.APIGroupLis
 	}
 	return false
 }
+
+func GenClusterList(clientSet karmadaclientset.Interface, clusters []string) ([]string, error) {
```

Comment:
I haven't started the review yet, I have a question, are both pull/push modes applicable?

Reply:
Yes，both pull and push modes are applicable.

Reply:
Thank you for reply, can you post some tests results in here?

Reply:
Of course~
```bash
➜  karmada git:(top) ✗ _output/bin/linux/amd64/karmadactl top node  
NAME                    CLUSTER   CPU(cores)   CPU%   MEMORY(bytes)   MEMORY%    
member1-control-plane   member1   360m         9%     804Mi           5%         
member2-control-plane   member2   362m         9%     810Mi           5%
member3-control-plane   member3   89m          2%     632Mi           3%
➜  karmada git:(top) ✗ ka get cluster                              
NAME      VERSION   MODE   READY   AGE 
member1   v1.27.3   Push   True    46m 
member2   v1.27.3   Push   True    46m
member3   v1.27.3   Pull   True    46m
```
As you can see, member1 is on push mode and member3 is on pull mode, and they are both applicable.